fuel-p2p/Cargo.toml

```
@@ -13,14 +13,19 @@ description = "Fuel client networking"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-trait = "0.1.52"
+bincode = "1.3"
 futures = "0.3"
 futures-timer = "3.0"
 ip_network = "0.4"
 libp2p = "0.41"
+rand = "0.8"
+serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.9"
+tokio = { version= "1.17", features = ["sync"] }
```

shouldn't this match the version of tokio below?

```suggestion
tokio = { version= "1.8", features = ["sync"] }
```